src/site/content/en/blog/nfc/index.md

```diff
@@ -0,0 +1,911 @@
+      break;
+    case 2:
+      // Open for editing
+      break;
+  }
+}
+```
+
+To write a smart poster record, pass an NDEF message to the NDEFWriter `write()`
+method. The smart poster record contained in the NDEF message is defined as an
+object with a `recordType` key set to `"smart-poster"` and a `data` key set to
+an object that represents (once again) an NDEF message contained in the
+smart-poster record.
+
+```js
+const encoder = new TextEncoder();
+const smartPosterRecord = {
+  recordType: "smart-poster",
+  data: {
+    records: [
+      {
+        recordType: "url", // URL record for smart poster content
+        data: "https://my.org/content/19911"
+      },
+      {
+        recordType: "text", // title record for smart poster content
+        data: "Funny dance"
+      },
+      {
+        recordType: ":t", // type record, a local type to smart poster
+        data: encoder.encode("image/gif") // MIME type of smart poster content
+      },
+      {
+        recordType: ":s", // size record, a local type to smart poster
+        data: new Uint32Array([4096]) // byte size of smart poster content
+      },
+      {
+        recordType: ":act", // action record, a local type to smart poster
+        // do the action, in this case open in the browser
+        data: new Uint8Array([0])
+      },
+      {
+        recordType: "mime", // icon record, a MIME type record
+        mediaType: "image/png",
+        data: await (await fetch("icon1.png")).arrayBuffer()
+      },
+      {
+        recordType: "mime", // another icon record
+        mediaType: "image/jpg",
+        data: await (await fetch("icon2.jpg")).arrayBuffer()
+      }
+    ]
+  }
+};
+
+const writer = new NDEFWriter();
+await writer.write({ records: [smartPosterRecord] });
+```
+
+### Read and write an external type record
+
+To create application defined records, use external type records. These may
+contain an NDEF message as payload that is accessible with `toRecords()`. Their
+name contains the domain name of the issuing organization, a colon and a type
+name that is at least one character long, for instance "example.com:foo".
+
+```js
+function readExternalTypeRecord(externalTypeRecord) {
+  for (const record of externalTypeRecord.toRecords()) {
+    if (record.recordType == "text") {
+      const decoder = new TextDecoder(record.encoding);
+      console.log(`Text: ${textDecoder.decode(record.data)} (${record.lang})`);
+    } else if (record.recordType == "url") {
+      const decoder = new TextDecoder();
+      console.log(`URL: ${decoder.decode(record.data)}`);
+    } else {
+      // TODO: Handle other type of records.
+    }
+  }
+}
+```
+
+To write an external type record, pass an NDEF message dictionary to the
+NDEFWriter `write()` method. The external type record contained in the NDEF
+message is defined as an object with a `recordType` key set to the name of the
+external type and a `data` key set to an object that represents (once again) an
```

Comment:
Delete '(once again)'. Many people are not going to read this from end to end. They'll use it as a reference work the same as a dictionary or an encyclopedia. You wouldn't find comments in those like "look, another noun" or "how many Prime Ministers are there?"

Reply:
Noted. Done.